github/codeowners.py

```diff
@@ -155,6 +156,19 @@ def determine_email_address(
     return EmailAddress(user.email)
 
 
+def usernames_with_email_address(
+    email_address: EmailAddress,
+    gh_api: GitHub | GitHubEnterprise,
+) -> typing.Generator[Username, None, None]:
+    '''
+    Return generator yielding usernames found for email addresse.
```

Comment:
this comment merely repeats the typehints - just omit it
also: address does not end with an `e` in english